test/Microsoft.ML.StaticPipelineTesting/Training.cs

```diff
@@ -38,7 +38,7 @@ public void SdcaRegression()
             var dataPath = GetDataPath(TestDatasets.generatedRegressionDataset.trainFilename);
             var dataSource = new MultiFileSource(dataPath);
 
-            var ctx = new RegressionContext(env);
+            var catalog = new RegressionCatalog(env);
```

Comment:
>new RegressionCatalog(env); [](start = 26, length = 27)

Hmmmm interesting. This guy is a bit of an odd duck, I guess it was done before we "unified" the contexts under `MLContext`. We'll probably want to bring these more up to date at some point lest they mislead people. (Not part of this PR, to be clear.)